packages/themes/src/tokens/v11TokenGroup.js

```diff
@@ -286,6 +286,7 @@ export const icon = TokenGroup.create({
       state: 'disabled',
       name: 'icon-disabled',
     },
+    'icon-inverse',
```

Comment:
should this be icon-interactive?
```suggestion
    'icon-interactive',
```

Reply:
yeah whoops! 🙃  Thanks for catching